[core]/es_extended/client/functions.lua

```diff
@@ -430,7 +430,11 @@ function ESX.Game.SpawnVehicle(vehicleModel, coords, heading, cb, networked)
         while not HasCollisionLoadedAroundEntity(vehicle) do
             Wait(0)
         end
-
+        if not IsVehicleOnAllWheels(vehicle) then
+		    local carCoords = GetEntityRotation(vehicle, 2)
+		    Wait(500)
+		    SetEntityRotation(vehicle, carCoords[1], 0, carCoords[3], 2, true)
```

Comment:
Is this 1-3 index all time?

Reply:
Yep, tried with all kind of vehicles (mostly addons trailers, boats and planes) and it works well

You can try with this trailer who spawn upside-down https://www.gta5-mods.com/vehicles/brute-camp-trailer-addon